blockchain/cryptoProvider/interface.go

```diff
@@ -0,0 +1,15 @@
+package cryptoProvider
+
+import (
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core"
+)
+
+// CryptoComponentsHolder is able to holder and provide all the crypto components
+type CryptoComponentsHolder interface {
+	GetPublicKey() crypto.PublicKey
+	GetPrivateKey() crypto.PrivateKey
+	GetBech32() string
```

Comment:
GetBech32Address?

Also, this interface should be moved where used, not implemented :D

Reply:
done